### ISSUE TITLE
emqx v4.2.0 UI frozen at the time when user navigating to client detail page

### ISSUE THREAD

**Issue by gsngsngsn** [emqx/emqx#3742]
**Environment**:

- EMQ X version (e.g. `emqx_ctl status`): 4.2.0
- Hardware configuration (e.g. `lscpu`): any
- OS (e.g. `cat /etc/os-release`): any
- Kernel (e.g. `uname -a`): any
- Erlang/OTP version (in case you build emqx from source code): 
- Others: chrome, edge browser

**What happened and what you expected to happen**:

WhatHappened:
1. Login to EMQX browser ui
2. GoTo the client page
3. click on specific clients (with certain client id: project%2F2rvk4e3gkdnl7u1kl0k%2Fclient%2Fclient%3DXXDfQA-ruQKfza9f%26scope%3Duser%2Csync%2Fuser%2F150159431%2Fdevice%2Fdiv101.e352eaeb1f0da67d4333df6a047fbbb7e170adaf13fe63d127e5099429d347bb )

Expected: navigate to client detail page
actual: page got frozen. 

**How to reproduce it (as minimally and precisely as possible)**:

I assume the problem lies on the clientId part, clientId: project/2rvk4e3gkdnl7u1kl0k/client/client=XXDfQA-ruQKfza9f&scope=user,sync/user/150159431/device/div101.e352eaeb1f0da67d4333df6a047fbbb7e170adaf13fe63d127e5099429d347bb

**Anything else we need to know?**:

1. Wroks fine with clientId (2ab5a772-2842-4f83-8a4b-6579355d447b)

```
curl 'https://emq.xbase.xyz/api/v4/clients/2ab5a772-2842-4f83-8a4b-6579355d447b?_=1601258948432' \
>   -H 'authority: emq.xbase.xyz' \
>   -H 'authorization: Basic YWRtaW46cHVibGlj' \
>   -H 'cache-control: no-cache' \
>   -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36' \
>   -H 'accept: */*' \
>   -H 'sec-fetch-site: same-origin' \
>   -H 'sec-fetch-mode: cors' \
>   -H 'sec-fetch-dest: empty' \
>   -H 'referer: https://emq.xbase.xyz/' \
>   -H 'accept-language: zh-CN,zh;q=0.9' \
>   --compressed
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   762  100   762    0     0  11373      0 --:--:-- --:--:-- --:--:-- 11373{"data":[{"keepalive":30,"mailbox_len":0,"mqueue_len":0,"awaiting_rel":0,"recv_cnt":1810,"send_oct":4066,"proto_ver":3,"clean_start":true,"clientid":"2ab5a772-2842-4f83-8a4b-6579355d447b","recv_pkt":220,"send_msg":0,"connected":true,"heap_size":1598,"subscriptions_cnt":0,"recv_oct":108251,"created_at":"2020-09-27 10:03:13","username":"super","connected_at":"2020-09-27 10:03:13","inflight":0,"proto_name":"MQIsdp","node":"xbase-emqx-emqx-chart@172.20.10.36","max_inflight":32,"ip_address":"172.20.21.108","max_subscriptions":0,"send_pkt":1813,"mqueue_dropped":0,"expiry_interval":0,"zone":"external","reductions":1106041,"send_cnt":1813,"is_bridge":false,"recv_msg":219,"max_awaiting_rel":100,"max_mqueue":1000,"port":34794,"mountpoint":"undefined"}],"code":0}
```

```
curl 'https://emq.xbase.xyz/api/v4/nodes/xbase-emqx-emqx-chart@172.20.10.36/subscriptions/2ab5a772-2842-4f83-8a4b-6579355d447b?_=1601258948538' \
>   -H 'authority: emq.xbase.xyz' \
>   -H 'authorization: Basic YWRtaW46cHVibGlj' \
>   -H 'cache-control: no-cache' \
>   -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36' \
>   -H 'accept: */*' \
>   -H 'sec-fetch-site: same-origin' \
>   -H 'sec-fetch-mode: cors' \
>   -H 'sec-fetch-dest: empty' \
>   -H 'referer: https://emq.xbase.xyz/' \
>   -H 'accept-language: zh-CN,zh;q=0.9' \
>   --compressed
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    20  100    20    0     0    294      0 --:--:-- --:--:-- --:--:--   294{"data":[],"code":0}

```

2. requesting the problematic client, 
Page frozed: no req fired 
![image](https://user-images.githubusercontent.com/3394919/94383543-00219600-0173-11eb-85c3-ee1c13ba20e0.png)

But the broker is responding fine when firing curl:

```
curl 'https://xxxxxxxx/api/v4/clients/project%2F2rvk4e3gkdnl7u1kl0k%2Fclient%2Fclient%3DXXDfQA-ruQKfza9f%26scope%3Duser%2Csync%2Fuser%2F150159431%2Fdevice%2Fdiv101.e352eaeb1f0da67d4333df6a047fbbb7e170adaf13fe63d127e5099429d347bb?_=1601201776780' \
>   -H 'authority: emq.xbase.xyz' \
>   -H 'authorization: Basic YWRtaW46cHVibGlj' \
>   -H 'cache-control: no-cache' \
>   -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36' \
>   -H 'accept: */*' \
>   -H 'sec-fetch-site: same-origin' \
>   -H 'sec-fetch-mode: cors' \
>   -H 'sec-fetch-dest: empty' \
>   -H 'accept-language: zh-CN,zh;q=0.9' \
  -H 'referer: https://emq.xbase.xyz/' \
>   -H 'accept-language: zh-CN,zh;q=0.9' \
>   --compressed
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   881  100   881    0     0  12408      0 --:--:-- --:--:-- --:--:-- 12585{"data":[{"keepa"awaiting_rel":0,"recv_cnt":2,"send_oct":29,"proto_ver":5,"clean_start":false,"clientid":"projfQA-ruQKfza9f&scope=user,sync/user/150159431/device/div101.e352eaeb1f0da67d4333df6a047fbbb7e17:2,"send_msg":0,"connected":true,"heap_size":4185,"subscriptions_cnt":3,"recv_oct":1090,"creat150159431","connected_at":"2020-09-27 10:06:11","inflight":0,"proto_name":"MQTT","node":"xbaseght":32,"ip_address":"183.2.223.26","max_subscriptions":0,"send_pkt":2,"mqueue_dropped":0,"expuctions":86411,"send_cnt":2,"is_bridge":false,"recv_msg":0,"max_awaiting_rel":100,"max_mqueue"d"}],"code":0}
```

**Comment by tigercl** [comment]
@gsngsngsn We are sorry that we did not reproduce your problem, even if the client identifier you gave has been used. We suggest you recheck the process, if the problem still exists, we will continue to troubleshoot.

**Comment by gsngsngsn** [comment]
> @gsngsngsn We are sorry that we did not reproduce your problem, even if the client identifier you gave has been used. We suggest you recheck the process, if the problem still exists, we will continue to troubleshoot.

Hi @tigercl , it is a 100% present case on my side right now:

![image](https://user-images.githubusercontent.com/3394919/94519967-10626f80-025e-11eb-8d25-1c80968205f1.png)
![image](https://user-images.githubusercontent.com/3394919/94520004-2112e580-025e-11eb-9dc7-84c13e5640fa.png)

**Comment by gsngsngsn** [comment]
seems like you fixed it @4.2.1 https://github.com/emqx/emqx-dashboard/pull/262
